enhancements/sig-architecture/30-clusterset-override/README.md

```diff
@@ -327,62 +327,40 @@ So we could finish the migration by four steps, and step 1 and step 2 will be fi
 1. [Implement in OCM 0.7.0]Update the managedClusterSet API which only includes an exclusive way to select target managedClusters.
 
 ```go
+// ManagedClusterSetSpec describes the attributes of the ManagedClusterSet
 type ManagedClusterSetSpec struct {
-    // Selector represents a selector of ManagedClusters by labels and names.
-    ClusterSelector ManagedClusterSelector  `json:"clusterSelector"`
+	// ClusterSelector represents a selector of ManagedClusters
+	// +optional
+	// +kubebuilder:default:={selectorType: LegacyClusterSetLabel}
+	ClusterSelector ManagedClusterSelector `json:"clusterSelector,omitempty"`
 }
 
-type ManagedClusterSelector struct{
-    // "" means to use the current mechanism of matching label <cluster.open-cluster-management.io/clusterset:<ManagedClusterSet Name>.
-    // (future) "LabelSelector" means to use the LabelSelector to select target managedClusters
-    // "ExclusiveLabel" means to use a particular cluster label. It is guaranteed that clustersets with same label key are exclusive with each others
-    // +optional
-    SelectorType SelectorType `json:"selectorType"`
-
-    // ExclusiveLabel defines one label which clusterset could use to select target managedClusters. In this way, we will:
-    // 1. Guarantee clustersets with same label key are exclusive
-    // 2. Enable additional permission check when cluster joining/leaving a clusterset (the label key should start with the reserved prefix "cluster.open-cluster-management.io/" and "info.open-cluster-management.io/");
-    ExclusiveLabel *ExclusiveLabel `json:"exclusiveLabel"`
+// ManagedClusterSelector represents a selector of ManagedClusters
+type ManagedClusterSelector struct {
+	// SelectorType could only be "LegacyClusterSetLabel" now, will support more SelectorType later
+	// "LegacyClusterSetLabel" means to use label "cluster.open-cluster-management.io/clusterset:<ManagedClusterSet Name>"" to select target clusters.
+	// +kubebuilder:validation:Enum=LegacyClusterSetLabel
+	// +kubebuilder:default:=LegacyClusterSetLabel
+	// +required
+	SelectorType SelectorType `json:"selectorType,omitempty"`
 }
 
 type SelectorType string
 
 const (
-    ExclusiveLabel        SelectorType = "ExclusiveLabel"
+	LegacyClusterSetLabel SelectorType = "LegacyClusterSetLabel"
 )
-
-//ExclusiveLabel defines one cluster label
-type ExclusiveLabel struct {
-    //Key is "cluster.open-cluster-management.io/clusterset" by default and can only be cluster.open-cluster-management.io/
-    Key string `json:"key"`
-    //Value can only be empty or the name of the clusterset.
-    Value string `json:"value"`
-}
 ```
 
-- `LabelSelector` will not be included
```

Comment:
let's keep the labelSelector since it should done in this release

Reply:
done